dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -466,16 +466,19 @@ public void launchWithArgs(final String entry, final String localEntry, final St
                         final String descriptor = descriptorType.getValue().toLowerCase();
                         LanguageClientInterface languageClientInterface = convertCLIStringToEnum(descriptor);
                         DescriptorLanguage language = DescriptorLanguage.convertShortStringToEnum(descriptor);
-
+                        if (workflow.getDefaultTestParameterFilePath() == null) {
+                            conditionalErrorMessage((yamlRun != null) == (jsonRun != null),
```

Comment:
If a user passes in values under both `--yaml` and `--json`, this will trigger. It's a niche case, but the error message might still be confusing. But, depending on the comment above, this might not be needed at all.